CONTRIBUTING.md

```diff
@@ -44,52 +50,45 @@ Before running this command make sure Docker is running. Read more about [tests]
 
-- Make a new folder with the name of the component under `src` folder.
-- Break variables and mixins into separate files where possible.
-- Make sure to include a `classes.scss` file in the folder as this file is used to generate all the relevant css classes.
+### Adding a new component
 
-  - This file should simply define classes using your mixins.
-  - Example from `src/toggle-switch/classes.scss`, where index imports all mixins and relevant scss.
-
-    ```scss
-    @import './index';
-
-    .iui-toggle-switch {
-      @include iui-toggle-switch;
-    }
-    ```
+If you'd like to get your component added to iTwinUI, follow these guidelines.
 
-  - Make sure to import your style in `src/classes.scss` and `src/mixins.scss`.
+We provide a script that can automatically create all the necessary files for you. Simply run `yarn createComponent [component-name]` (e.g. `yarn createComponent my-component`).
 
-- Write html for your new component in `your-component.html` file under `backstop/tests`
+- Add all your component styles in `src/[component-name]/[component-name].scss` file.
+  - Break variables and mixins into separate files where possible.
+- Define classes for your mixin in `src/[component-name]/classes.scss` file.
+  - *Running the `createComponent` command will do this for you.*
+- Make sure your component index and classes are imported in `src/index.scss` and `src/classes.scss`.
+  - *Running the `createComponent` command will also do this for you but you need to manually sort the imports alphabetically.*
+- Write tests for your new component in `backstop/tests/[component-name].html` and `backstop/scenarios/[component-name].js`. See [Testing](#Testing) section below.
 - After running `yarn build` you can open minified html in browser to check up, how your component looks like from `backstop/minified`.
 
-### Editing or enhancing a component
+### Testing
 
-When making changes to a component that already exists, we ask that all PRs distinctly list visual changes that have occurred. These notes are used by the Visual Design team to update specifications and images within documentation.
+For running tests you will need [Docker](https://www.docker.com/products/docker-desktop). It helps to avoid cross-platform rendering differences.
 
-### Tests
+- To run tests for a specific component, use this command:
 
-For testing you will need [Docker](https://www.docker.com/products/docker-desktop). It helps to avoid cross-platform rendering differences.
+  `yarn test --filter=[component_name]` (e.g. `yarn test --filter=side-navigation`)
 
-To run selected tests use command:
+- To approve test images, run `yarn approve`.
 
- `yarn test --filter="test_name"`
+- To delete old/unused tests images, run `yarn clean:images`.
 
-To approve tests use command:
+#### How to write tests:
 
-`yarn approve`
+- Create `.html` file in `backstop/tests` displaying the elements you wish to test and their all possible states, using the built CSS in `lib/css/`.
```

Comment:
I think file will be created already with script. So just need to write the html in there (start with display the element, i.e.)